File: Iniciante/1012- Área/Python/1012.py
def main():
    A, B, C = input().split()
    A = float(A)
    B = float(B)
    C = float(C)
    pi = 3.14159

    print(f"TRIANGULO: {area_triangulo(A, C):.3f}")
    print(f"CIRCULO: {area_circulo(pi, C):.3f}")
    print(f"TRAPEZIO: {area_trapezio(A, B, C):.3f}")
    print(f"QUADRADO: {area_quadrado(B):.3f}")
    print(f"RETANGULO: {area_retangulo(A, B):.3f}")


def area_triangulo(a, c):
    return a * c / 2


def area_circulo(p, c):
    return p * c**2


def area_trapezio(a, b, c):
    return ((a + b) * c) / 2


def area_quadrado(b):
    return b * b


def area_retangulo(a, b):
    return a * b


main()

File: Iniciante/1012- Área/JavaScript/1012.js
function main() {
  var input = require("fs").readFileSync("/dev/stdin", "utf8");
  var lines = input.split(" ");

  const A = parseFloat(lines[0]);
  const B = parseFloat(lines[1]);
  const C = parseFloat(lines[2]);
  const pi = 3.14159;

  console.log(`TRIANGULO: ${areaTriangulo(A, C).toFixed(3)}`);
  console.log(`CIRCULO: ${areaCirculo(pi, C).toFixed(3)}`);
  console.log(`TRAPEZIO: ${areaTrapezio(A, B, C).toFixed(3)}`);
  console.log(`QUADRADO: ${areaQuadrado(B).toFixed(3)}`);
  console.log(`RETANGULO: ${areaRetangulo(A, B).toFixed(3)}`);
}

function areaTriangulo(a, c) {
  return (a * c) / 2;
}

function areaCirculo(p, c) {
  return p * c ** 2;
}

function areaTrapezio(a, b, c) {
  return ((a + b) * c) / 2;
}

function areaQuadrado(b) {
  return b * b;
}

function areaRetangulo(a, b) {
  return a * b;
}

main();
